luci/luci-app-ffwizard-falter/luasrc/tools/freifunk/assistent/tools.lua

```diff
@@ -4,6 +4,20 @@ local uci = require "luci.model.uci".cursor()
 
 module("luci.tools.freifunk.assistent.tools", package.seeall)
 
+function wifi_get_mesh_modes(device)
+	local modes = {}
+        local phy = string.gsub(device, "radio", "phy")
+	local iwOutput = sys.exec("iw phy "..phy.." info | grep -A1 \"valid interface combination\" | tail -1")
+        if string.find(iwOutput, "mesh point") then
```

Comment:
From here on it looks a bit unusual in concern of identation.

Reply:
Maybe just passing the stuff to vs code and do [crl] + [shift] + [i]... xD